src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1050,23 +1169,6 @@ public bool IsTopMostNode
             }
         }
 
-        internal void ConnectInput(int inputData, int outputData, NodeModel node)
```

Comment:
This is as yet completely untested, but after looking at the referenced function, I thought about ways that it might be made more efficient.
https://github.com/DynamoDS/Dynamo/commit/f92af272ba12faa449b7d7c6965e1e03f943ccc6